pinot-segment-local/src/test/java/org/apache/pinot/segment/local/recordtransformer/RecordTransformerTest.java

```diff
@@ -178,6 +178,29 @@ public void testSanitationTransformer() {
     }
   }
 
+  @Test
+  public void testLogicalScalarOps() {
+    TableConfig tableConfig = new TableConfigBuilder(TableType.OFFLINE).setTableName("testTable").build();
+
+    // expression true, filtered
+    GenericRow genericRow = getRecord();
+    tableConfig.setIngestionConfig(
+        new IngestionConfig(null, null,
+            new FilterConfig("AND(EQUALS(svInt, 123), LESS_THAN_OR_EQUAL(svDouble, 200))"), null, null));
```

Comment:
Let's change the expression to `svInt = 123 AND svDouble <= 200`. Users won't explicitly write query in this format

Reply:
ack